app/views/backoffice/audit/index.en.html.erb

```diff
@@ -1,27 +1,25 @@
 <% title 'Back office: audit records' %>
 
-<div class="grid-row">
-  <div class="column-full">
-    <h1 class="heading-xlarge gv-u-heading-xxlarge">Back office: audit records</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl">Back office: audit records</h1>
 
-    <p class="heading-medium">
-      Actions performed in the last 30 days
-    </p>
+    <h3 class="govuk-heading-m"> Actions performed in the last 30 days</h3>
 
-    <table class="backoffice-table">
```

Comment:
We still need to maintain some of the custom styling. Migrating where possible to new classes but if not possible, as with the check your answer, we use custom styles:
app/assets/stylesheets/local/app/backoffice.scss

Reply:
Do we want to keep the data smaller on purpose? I don't see much difference other than font size, but I don't know the reason behind that decision

Reply:
The smaller size is for the show details content, but it is not that important. The main thing would be the no-border so there is no extra border below like this:

<img width="615" alt="Screen Shot 2020-05-05 at 08 43 42" src="https://user-images.githubusercontent.com/687910/81044711-c17dfd00-8eac-11ea-90b5-96dd2ce624e1.png">

If there are many audit entries, that extra border add a lot of noise and clutter.

Reply:
![image](https://user-images.githubusercontent.com/136777/81067206-374a8e80-8ed6-11ea-9a32-b0c022a82f6b.png)

Reply:
👍